sparkpost/transmissions.py

```diff
@@ -265,15 +266,18 @@ def get(self, transmission_id):
         results = self._fetch_get(transmission_id)
         return results['transmission']
 
-    def list(self):
+    def list(self, **kwargs):
         """
         Get a list of your transmissions
 
+        :param campaign_id: ID of the campaign used by the transmissions
+        :param template_id: ID of the template used by the transmissions
+
         :returns: list of transmissions
         :raises: :exc:`SparkPostAPIException` if API call fails
         """
-        results = self.request('GET', self.uri)
-        return results
+        warnings.warn('deprecated', DeprecationWarning)
```

Comment:
I put a giant message initially but seemed too much. something like
```
This endpoint is deprecated. For details, check https://developers.sparkpost.com/api/transmissions.html#transmissions-list-get
```

To me it seemed verbose. But if you find this (or modify as you want) helpful, I can put that.